cirq-google/cirq_google/devices/serializable_device.py

```diff
@@ -108,6 +108,25 @@ def __init__(
         """
         self.qubits = qubits
         self.gate_definitions = gate_definitions
+        self._metadata = cirq.GridDeviceMetadata(
+            [
+                (pair[0], pair[1])
```

Comment:
nit: Also use keyword arguments in the `GridDeviceMetadata` constructor for clarity.

Reply:
Done.